### PR TITLE
Update Gretel to v0.0.92

### PR DESCRIPTION
**Pull request by SamStudio8**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/gretel/meta.yaml [hunk 1]
```diff
@@ -5,19 +5,19 @@ package:
...
+  noarch: python
 
 source:
   url: https://github.com/SamStudio8/gretel/archive/v{{ version }}.tar.gz
   sha256: 1b111e1c56944e8f712fd1aada12b450cb97e9a196437a6c374ebbe51b9209b1
 
 requirements:
   host:
-    - python
+    - python <3
```
Comment (SamStudio8) [inline recipes/gretel/meta.yaml]: gretel works with python3, do I need the `<3` here?

**Comment by SamStudio8** [comment]
@tseemann I think I've done 1 and 2 (I haven't maintained a conda yaml before). I can do 3 once I've bundled the tests for the package properly on pypi, but that'll take a version bump anyway. I'd like to leave 4 as it is for now because Gretel relies on specific versions of the Hansel graph API.

**Comment by tseemann** [comment]
```
12:36:59 BIOCONDA INFO (OUT) usage: gretel [-h] [-s START] [-e END] [-l LORDER] [-p PATHS]
12:36:59 BIOCONDA INFO (OUT)               [--master MASTER] [--gapchar GAPCHAR] [--delchar DELCHAR]
12:36:59 BIOCONDA INFO (OUT)               [--quiet] [-o OUT] [-@ THREADS] [--debugreads DEBUGREADS]
12:36:59 BIOCONDA INFO (OUT)               [--debugpos DEBUGPOS] [--debughpos DEBUGHPOS]
12:36:59 BIOCONDA INFO (OUT)               [--dumpmatrix DUMPMATRIX] [--dumpsnps DUMPSNPS]
12:36:59 BIOCONDA INFO (OUT)               bam vcf contig
12:36:59 BIOCONDA INFO (OUT) gretel: error: too few arguments
12:37:00 BIOCONDA INFO (OUT) Tests failed for gretel-0.0.92-py_0.tar.bz2 - moving package to /opt/conda/conda-bld/broken
12:37:00 BIOCONDA INFO (OUT) WARNING:conda_build.build:Tests failed for gretel-0.0.92-py_0.tar.bz2 - moving package to /opt/conda/conda-bld/broken
```

**Comment by tseemann** [comment]
It seems `gretel -h` returns a non-zero error code?

File: recipes/gretel/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/gretel/meta.yaml]: No, remove it

**Comment by SamStudio8** [comment]
Not quite sure why this is failing now. There is a small set of unit tests in the 0.0.93 package that can be run with `python -m unittest` in the package root, but I'm not sure how to specify this properly.

**Comment by dpryan79** [comment]
Thanks @SamStudio8!

**Comment by SamStudio8** [comment]
Ah but I implemented it after the first failure. I could see it running in
the CircleCI log.

14:28:37 BIOCONDA INFO (OUT)   --version             show program's
version number and exit
14:28:37 BIOCONDA INFO (OUT) + gretel --version
14:28:37 BIOCONDA INFO (OUT) + grep 'gretel 0.0.93'
14:28:37 BIOCONDA INFO (OUT) gretel 0.0.93
14:28:39 BIOCONDA INFO (OUT) Tests failed for
gretel-0.0.93-py_0.tar.bz2 - moving package to
/opt/conda/conda-bld/broken
14:28:39 BIOCONDA INFO (OUT) WARNING:conda_build.build:Tests failed
for gretel-0.0.93-py_0.tar.bz2 - moving package to
/opt/conda/conda-bld/broken
14:28:39 BIOCONDA INFO (OUT) TESTS FAILED: gretel-0.0.93-py_0.tar.bz2
14:28:41 BIOCONDA ERROR COMMAND FAILED (exited with 1): docker run -t
--net host --rm -v
/tmp/tmpib5sbdt8/build_script.bash:/opt/build_script.bash -v
/home/circleci/project/miniconda/conda-bld/:/opt/host-conda-bld -v
/home/circleci/project/recipes/gretel:/opt/recipe -e LANG=en_US.UTF-8
-e HOST_USER_ID=1001 bioconda/bioconda-utils-build-env:latest
/bin/bash /opt/build_script.bash

**Pull request merged**